packages/admin-ui/src/pages/packages/components/Network/HttpsMappings.tsx

```diff
@@ -107,6 +107,19 @@ export function HttpsMappings({
     }
   }
 
+  if (mappings.error && mappings.error.message.includes("ENOTFOUND"))
```

Comment:
This error detection is too broad. The DAPPMANAGER should be able to detect that the HTTPs portal is not installed and communicate that to the Admin UI in a clear way